ERCS/erc-7590.md

```diff
@@ -0,0 +1,402 @@
+---
+eip: 7590
```

Comment:
```suggestion
eip: 7584
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
Will do, thank you